pynicotine/gtkgui/widgets/iconnotebook.py

```diff
@@ -408,7 +408,7 @@ def update_pages_menu_button(self):
 
         if self.unread_pages:
             icon_name = "emblem-important-symbolic"
-            tooltip_text = _("Unread Tabs")
+            tooltip_text = f'{len(self.unread_pages)} {_("Unread Tabs")}'
```

Comment:
```suggestion
            tooltip_text = _("%(num)i Unread Tabs") % {"num": len(self.unread_pages)}'
```

Reply:
Are you sure it's worth creating a new translatable string for this?

Reply:
Might also be worth looking into using format() for translatable strings in the future. You could simply do `_("{num} Unread Tabs").format(num=len(self.unread_pages))`, and the strings would look cleaner.

Reply:
> creating a new translatable string, it's safer to postpone this change

In that case, I will mark this PR as draft.

Reply:
Since we're still doing final tweaks to strings, I decided to add a new string: 963370ddf3b22f16825eb5dcaf2df5e6c2d734ac

I'll of course give more time for translators to complete their translations.